src/YaPI.pm.in

```diff
@@ -26,6 +26,9 @@ our @EXPORT = qw(textdomain __);
 use YaST::YCP;
 use ycp;
 
+# FATE#323734
+y2warning("YaPI is deprecated therefore this is not a stable API to be used by external resources anymore. It will be removed completely in a close future.");
```

Comment:
IMHO too long message, something like `YaPI is deprecated, do not use anymore! It will be removed soon.`